web/src/locales/zh/data.json

```diff
@@ -324,7 +324,17 @@
     "Users under all organizations": "所有组织里的用户",
     "Webhooks": "Webhooks",
     "empty": "无",
-    "{total} in total": "{total} 总计"
+    "{total} in total": "{total} 总计",
```

Comment:
This i18n is auto-generated: https://casdoor.org/docs/contributing#how-to-update-i18n-data